src/camvidlog2/moondream/cli.py

```
@@ -0,0 +1,88 @@
+        if existing_array is None:
+            existing_array = array
+        else:
+            existing_array = pd.concat([existing_array, array])
+
+        # save array (new or existing) to disk
+        existing_array.to_feather(db)
+        print("saved file")
+
+
+@app.command()
+def show(
```

Ruff (0.12.2)

Missing return type annotation for public function `show`

Add return type annotation: `None`

(ANN201)